content/en/docs/admin/admission-controllers.md

```diff
@@ -52,35 +54,27 @@ support all the features you expect.
 
 ## How do I turn on an admission controller?
 
-The Kubernetes API server flag `enable-admission-plugins` takes a comma-delimited list of admission control plugins to invoke prior to modifying objects in the cluster.
-For example, the following command line enables the `NamespaceLifecycle` and the `LimitRanger`
-admission control plugins:
+The Kubernetes API server supports a flag, `admission-control` that takes a comma-delimited,
+ordered list of admission control choices to invoke prior to modifying objects in the cluster.
+For example, the following command line turns on the `NamespaceLifecycle` and the `LimitRanger`
+admission controller:
 
 ```shell
-kube-apiserver --enable-admission-plugins=NamespaceLifecyle,LimitRanger ...
+kube-apiserver --admission-control=NamespaceLifecyle,LimitRanger ...
 ```
 
-{{< note >}}
 **Note**: Depending on the way your Kubernetes cluster is deployed and how the
 API server is started, you may need to apply the settings in different ways.
 For example, you may have to modify the systemd unit file if the API server is
 deployed as a systemd service, you may modify the manifest file for the API
 server if Kubernetes is deployed in a self-hosted way.
-{{< /note >}}
-
-## How do I turn off an admission controller?
-
-The Kubernetes API server flag `disable-admission-plugins` takes a comma-delimited list of admission control plugins to be disabled, even if they are in the list of plugins enabled by default.
-
-```shell
-kube-apiserver --disable-admission-plugins=PodNodeSelector,AlwaysDeny ...
-```
+{: .note}
```

Comment:
Please modify this to use the Hugo short-code. See https://kubernetes.io/docs/test/#admonitions or its source code at https://raw.githubusercontent.com/kubernetes/website/master/content/en/docs/test.md.